bucket/win-dynamic-desktop.json

```
@@ -0,0 +1,32 @@
+{
+    "version": "3.3.0",
+    "homepage": "https://github.com/t1m0thyj/WinDynamicDesktop",
+    "description": "Cycle through multiple wallpapers based on the times of sunrise and sunset.",
+    "license": "MPL-2.0",
+    "url": "https://github.com/t1m0thyj/WinDynamicDesktop/releases/download/v3.3.0/WinDynamicDesktopPortable.exe#/WinDynamicDesktop.exe",
+    "hash": "3478c32c24caf1811d3a7039227c9497a7e35aaed445deca09849a540faf8b4c",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\WinDynamicDesktop.pth\")) {",
+        "    New-Item \"$dir\\WinDynamicDesktop.pth\" -Value \"$(current_dir $dir)\" | Out-Null",
+        "}",
+        "if (!(Test-Path \"$persist_dir\\settings.conf\")) {",
```

One line is 120 chars long. Github PR is 127 and github code view is 140.

```suggestion
        "if (!(Test-Path \"$persist_dir\\settings.conf\")) { New-Item \"$dir\\settings.conf\" -Value '{}' | Out-Null }"
```